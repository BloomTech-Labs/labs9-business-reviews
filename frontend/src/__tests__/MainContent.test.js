import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';
import MainContentComponent from '../components/MainContent';

describe('MainContentComponent />', () => {
  it('renders', () => {
    shallow(<MainContentComponent />);
  });
  it('matches snapshot', () => {
    const wrapper = shallow(<MainContentComponent />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
  it('should contain PopularBusinesses and PopularReviewers', () => {
    let main = shallow(<MainContentComponent />);
    expect(main.find("PopularBusinesses").length).toBe(1);
    expect(main.find("PopularReviewers").length).toBe(1);
  })
});
