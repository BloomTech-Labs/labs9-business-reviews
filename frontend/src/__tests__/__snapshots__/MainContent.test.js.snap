// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContentComponent /> matches snapshot 1`] = `
<styled.div>
  <PopularBusinesses />
  <styled.div>
    <PopularReviewers />
  </styled.div>
</styled.div>
`;
